base/Dockerfile

```diff
@@ -13,22 +13,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG tag
 
 FROM adoptopenjdk/openjdk11:alpine AS base
 ARG version
 RUN echo "Building Apache SkyWalking ${version}"
 ENV SKYWALKING_VERSION=${version}
-
-FROM base AS es6
-RUN echo "This is the stage that build the es6 image"
 ENV SKYWALKING_TGZ=apache-skywalking-apm-${SKYWALKING_VERSION}.tar.gz
 
-FROM base AS es7
-RUN echo "This is the stage that build the es7 image"
-ENV SKYWALKING_TGZ=apache-skywalking-apm-es7-${SKYWALKING_VERSION}.tar.gz
-
-FROM ${tag} AS build
+FROM base AS build
```

Comment:
A tip, with these changes are not backward compatible, we need another archive to keep all these documentation and scripts, right?
We don't do official release for this repo.

Reply:
That another thing I want to ask, why don't we put these docker files in their corresponding repo so that they can evolve with the codebase instead of using manual `archive` here. Also they can be tested if they are in the corresponding repo (like oap in apache/skywalking and java agent in apache/skywalking-java) in the CI an PR process

Reply:
This is caused by ASF release process. Because in the old days, we need to change script for every release(tag name or some parameters), because this repo exists for years even before we support all configurations through system environments. So, you could image how this complex every time this docker push is.

Actually, for the current status, we don't need a new script, we just need a tag or commit id as import to package and push the image. It is reasonable for me to consider achieving this repo from now, and establish all things in those repositories you mentions. 

Once you decide to go into this directly, please send a notification(Lazy Consensus) mail to the dev ml. We could directly work in your proposed way.

Reply:
@kezhenxu94 Another tip, we need some update at here, https://github.com/apache/skywalking-kubernetes